example-2/evm/src/contracts/lendingHub/Hub.sol

```diff
@@ -53,11 +58,20 @@ contract Hub is HubStructs, HubMessages, HubGetters, HubSetters, HubUtilities {
 
         allowAsset(assetAddress);
 
-        AssetInfo memory info = AssetInfo({
-            collateralizationRatio: collateralizationRatio,
+        InterestRateModel memory interestRateModel = InterestRateModel({
+            ratePrecision: 1 * 10**18,
+            rateIntercept: 0,
```

Comment:
Do we want to hardcode every asset's interest rate to be constant at 0 (this is what fixing the rate coefficient and intercept at 0 does right)?

Reply:
No, those should be added as params eventually. Feel free to add these as params in this branch